src/components/run-not-found-warning/run-not-found-warning.js

```diff
@@ -0,0 +1,25 @@
+import { connect } from 'react-redux';
```

Comment:
I would keep this within workflow folder instead of having a new component folder

Reply:
makes sense maybe after this we should decide a consistent approach for all I would prefer very small components to be nested if they belong to a single component